### ISSUE TITLE
Sending images.

### ISSUE THREAD

**Issue by roccomuso** [eclipse/paho.mqtt.android#182]
Is there a way to send an image over mqtt using this lib?

**Comment by jpwsutton** [comment]
Definitely, MQTT is payload agnostic, so all you need to do is convert the image to a byte array and then publish it over MQTT. Your subscriber then would just have to take the Byte array and reassemble it into the image.
